draft-ietf-taps-transport-security.md

```diff
@@ -239,71 +253,19 @@ between connection instances.
-
-- Cryptographic algorithm negotiation: Negotiating support of protocol algorithms, including algorithms for
-encryption, hashing, MAC (PRF), and digital signatures.
-
-- Session caching and management: Managing session state caches used for subsequent connections,
-with the aim of amortizing connection establishment costs.
-
-- Peer authentication: Authenticating peers using generic or protocol-specific mechanisms, such as
-certificates, raw public keys, pre-shared keys, or EAP methods.
-
-- Unilateral responder authentication: Requiring authentication for the responder of a connection.
-
-- Mutual authentication: Establishing connections in which both endpoints are authenticated.
-
-- Application authentication delegation: Delegating to applications out-of-band to perform
-peer authentication.
-
-- Record (channel or datagram) confidentiality and integrity: Encrypting and authenticating
-application plaintext bytes sent between peers over a channel or in individual datagrams.
-
-- Partial record confidentiality: Encrypting some portion of records.
-
-- Optional record integrity: Optionally authenticating certain records.
-
-- Record replay prevention: Detecting and defending against record replays, which can be due
-to in-network retransmissions.
-
-- Early data support: Transmitting application data prior to secure connection
-establishment via a handshake. For TLS, this support begins with TLS 1.3.
-
-- Connection mobility: Allowing a connection to be multihomed or resilient across network
-interface or address changes, such as NAT rebindings that occur without an endpoint's knowledge. Mobility allows
-cryptographic key material and other state information to be reused in the event of a connection change.
-
-- Application-layer feature negotiation: Securely negotiating application-specific functionality.
-Such features may be necessary for further application processing, such as the TLS parent connection
-protocol type via ALPN {{?RFC7301}} or desired application identity via SNI {{?RFC6066}}.
-
-- Configuration extensions: Adding protocol features via extensions or configuration options. TLS
-extensions are a primary example of this feature.
-
-- Out-of-order record receipt: Processing of records received out-of-order.
-
-- Source validation (cookie or puzzle based): Validating peers and mitigating denial-of-service (DoS) attacks
-via explicit proof of origin (cookies) or work mechanisms (puzzles).
-
-- Length-hiding padding: Adding padding to records in order to hide plaintext message length
-and mitigate amplification attack vectors.
-
 # Transport Security Protocol Descriptions
 
-This section contains descriptions of security protocols currently used to protect data
-being sent over a network.
+This section contains brief descriptions of the various security protocols currently used to protect data
+being sent over a network. The interfaces between these protocols and transports is described
```

Comment:
s/is/are/ in both this and the next line. "interfaces (subordinate clause...) are"

Reply:
Fixed!